pnpm-lock.yaml

```diff
@@ -10844,3 +10840,7 @@ packages:
 
   /zod@3.21.4:
     resolution: {integrity: sha512-m46AKbrzKVzOzs/DZgVnG5H55N1sv1M8qZU3A8RIKbs3mrACDNeIOeilDymVb2HdmP8uwshOCF4uJ8uM9rCqJw==}
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
Is this configured by us?

Reply:
I am unsure it was automatically added, when I did `pnpm install`